assoc_space/__init__.py

```diff
@@ -409,31 +409,37 @@ def most_similar_to_vector(self, vec):
         terms_similar_to_vector.
         """
         sim = self.assoc.dot(vec)
-        index = np.argmax(self.assoc.dot(vec))
+        index = np.argmax(sim)
         return self.labels[index], sim[index]
 
-    def terms_similar_to_vector(self, vec, sorted=True, filter=None, num=None):
+    def terms_similar_to_vector(self, vec, sorted=True, filter=None, num=20):
         """
-        Find the terms most similar to the given vector, returning a list of
-        (term, similarity) tuples in descending order of similarity.  (Set
-        `sorted` to False to skip sorting.)
+        Find the `num` most similar terms to the given vector, returning a list
+        of (term, similarity) tuples.
+
+        If `sorted` is True, the list is in descending order of similarity.
+
+        If `filter` is given, the list will only contains terms that pass the
+        filter.
+
+        If `num` is None, then the list will contain all terms in the
+        assoc space.
         """
-        if num is not None and sorted:
+        # num < self.assoc.shape / 2 is from emperical testing
```

Comment:
"empirical", and you should probably elaborate that what's being tested is which version is faster.